src/main/java/blusunrize/immersiveengineering/common/blocks/metal/ChuteTileEntity.java

```diff
@@ -384,4 +388,72 @@ public boolean receiveClientEvent(int id, int arg)
 		return false;
 	}
 
+	private LazyOptional<IItemHandler> insertionCap = registerCap(() -> new ChuteTileEntity.ChuteInventoryHandler(this));
+
+	@Nonnull
+	@Override
+	public <T> LazyOptional<T> getCapability(@Nonnull Capability<T> cap, @Nullable Direction side)
+	{
+		if(cap==CapabilityItemHandler.ITEM_HANDLER_CAPABILITY&&side==Direction.UP)
+			return insertionCap.cast();
+		return super.getCapability(cap, side);
+	}
+
+	public static class ChuteInventoryHandler implements IItemHandlerModifiable
+	{
```

Comment:
This doesn't need to be Modifiable. Make it `IItemHandler` instead and ditch the `setStackInSlot` method

Reply:
done. This was almost a direct copy of ConveyorInventoryHandler. I don't know if that could be changed, too, or if there was a good reason for it to be IItemHandlerModifiable